pkg/cmd/options.go

```diff
@@ -53,7 +55,14 @@ func ControllerSwitchOptions() *controllercmd.SwitchOptions {
 // WebhookSwitchOptions are the webhookcmd.SwitchOptions for the provider webhooks.
 func WebhookSwitchOptions() *webhookcmd.SwitchOptions {
 	return webhookcmd.NewSwitchOptions(
-		webhookcmd.Switch(extensioncontrolplanewebhook.WebhookName, controlplanewebhook.AddToManager),
-		webhookcmd.Switch(extensioncontrolplanewebhook.ExposureWebhookName, controlplaneexposurewebhook.AddToManager),
+		webhookcmd.Switch(extensioncontrolplanewebhook.WebhookName, controlplanewebhook.New),
+		webhookcmd.Switch(extensioncontrolplanewebhook.ExposureWebhookName, controlplaneexposurewebhook.New),
+	)
+}
+
+// GardenWebhookSwitchOptions are the webhookcmd.SwitchOptions for the provider webhooks.
+func GardenWebhookSwitchOptions() *webhookcmd.SwitchOptions {
```

Comment:
Would it make sense to put this in a separate `pkg/validator/cmd` package?

Reply:
Hm good question but at least `pkg/validator/cmd` won't be adequate if we add the mutating/defaulting webhook.

Reply:
What do you mean? I was talking about the `GardenWebhookSwitchOptions` function that, according to my understanding, is only relevant for the validator binary, or? Most things related to the validator binary are already separated in `pkg/validator`, so I was thinking about introducing this function in `pkg/validator/cmd/options.go`.

Reply:
Yes, my thought was a bit forward. In the near future the validator will not be a validator but also a defaulter (both can be disabled by command line flag). Or is your suggestion to have another binary for defaulting provider specific parts of resources in the Garden cluster? My impression is that this would be a bit too undersized.

Reply:
Ah, understood now. No, I don't suggest a separate binary. Probably the name "validator" doesn't make sense anymore in this case - it should rather be called "admission-controller" or "admission-server" or similarly then. However, this renaming should be done once such defaulting webhooks are introduced, it's not needed now. I still think that for the current state it'd be slightly cleaner to separate these switch options to the `validator` package, so later (when "re-branded" to `admission` or similar) it is already at the right place and can be extended with switch options for defaulters.

Reply:
Thanks for the feedback. I've moved some packages and renamed some files to the more generic term `admission`, right away (see 80dc11886044f203733fa025a7c8fb0c559953ef). I'm already working on a defaulter but would prefer to open another PR for it for the sake of the reviewers.